packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -771,6 +771,68 @@ void main() {
     expect(find.text('BottomSheet'), findsNothing);
   });
 
+  testWidgets('the framework do not takeover the transitionAnimationController provided by user.', (WidgetTester tester) async {
```

Comment:
takeover => dispose

I understand what you mean by "takeover" however it doesn't hurt to be a little more explicit

Reply:
Please insert:
```
// Regression test for https://github.com/flutter/flutter/issues/87708
```